training/finetune_acft_hf_nlp/environments/acpt/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/acft-hf-nlp-gpu
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+    - requirements.txt
+  publish:
+    location: mcr
+    visibility: unlisted
```

Comment:
any reason to make it unlisted?

Reply:
This environment is specifically for finetune components and as such will not be used in general for other tasks. That is why we kept it in unlisted